tests/test_netbox.py

```diff
@@ -12,46 +12,37 @@
 VERSIONS = ["2.3.5", "2.8.9"]
 
 
-def get_inv(requests_mock, plugin, pagination, version, include_vms=False, **kwargs):
+def _create_mock(requests_mock, pagination, version, application, resource, **kwargs):
+    """initialises mock objects for testcase"""
     if not pagination:
-        with open(f"{BASE_PATH}/mocked/{version}/devices.json", "r") as f:
+        with open(f"{BASE_PATH}/mocked/{version}/{resource}.json", "r") as f:
             requests_mock.get(
-                "http://localhost:8080/api/dcim/devices/?limit=0",
+                f"http://localhost:8080/api/{application}/{resource}/?limit=0",
                 json=json.load(f),
                 headers={"Content-type": "application/json"},
             )
     else:
         for offset in range(3):
-            with open(f"{BASE_PATH}/mocked/{version}/devices-{offset}.json", "r") as f:
-                url = "http://localhost:8080/api/dcim/devices/?limit=0"
+            with open(
+                f"{BASE_PATH}/mocked/{version}/{resource}-{offset}.json", "r"
+            ) as f:
+                url = f"http://localhost:8080/api/{application}/{resource}/?limit=0"
                 requests_mock.get(
                     f"{url}&offset={offset}" if offset else url,
                     json=json.load(f),
                     headers={"Content-type": "application/json"},
                 )
-    return plugin(include_vms=include_vms).load()
 
 
-def get_vm_inv(requests_mock, plugin, pagination, version, **kwargs):
-    if not pagination:
-        with open(f"{BASE_PATH}/mocked/{version}/vms.json", "r") as f:
-            requests_mock.get(
-                "http://localhost:8080/api/virtualization/virtual-machines/?limit=0",
-                json=json.load(f),
-                headers={"Content-type": "application/json"},
-            )
-    else:
-        for offset in range(3):
-            with open(f"{BASE_PATH}/mocked/{version}/vms-{offset}.json", "r") as f:
-                url = (
-                    "http://localhost:8080/api/virtualization/virtual-machines/?limit=0"
-                )
-                requests_mock.get(
-                    f"{url}&offset={offset}" if offset else url,
-                    json=json.load(f),
-                    headers={"Content-type": "application/json"},
-                )
-    return get_inv(requests_mock, plugin, pagination, version, include_vms=True)
+def get_inv(requests_mock, plugin, pagination, version, **kwargs):
+
+    _create_mock(requests_mock, pagination, version, "dcim", "devices")
+    if "include_vms" in kwargs and kwargs["include_vms"]:
```

Comment:
`kwargs.get("include_vms", False)`

Reply:
Meant to do that.  :)  Still hanging on to some old methods.